src/index.js

```diff
@@ -16,7 +16,7 @@ let ignoreRegex;
 
 export default ({ types: t }) => ({
   visitor: {
-    ImportDeclaration(path, state) {
+    ImportDeclaration (path, state) {
```

Comment:
Let's leave this part as-is to align with the [Airbnb Styleguide](https://github.com/airbnb/javascript)